cargo-paradedb/src/bench_hits.rs

```diff
@@ -0,0 +1,345 @@
+    } else {
+        eprintln!("Invalid workload: {}", workload);
+        std::process::exit(1);
+    }
+
+    run_cmd!(echo "\nLoading dataset...")?;
+    env::set_var("PGPASSWORD", "postgres");
+
+    let conn_opts = &PgConnectOptions::from_str(url)?;
+    let mut conn = PgConnection::connect_with(conn_opts).await?;
+
+    let file_path = if workload == "single" {
+        single_file_path
+    } else {
+        partitioned_dir_path
+    };
+
+    println!("Creating foreign table from '{}'", file_path);
+
+    conn.execute(create_query_sql(&file_path).as_str())
+        .await
+        .unwrap();
+
+    run_cmd!(echo "\nRunning queries...")?;
+    run_queries(&mut conn).await?;
+
+    // run_cmd!(echo "\nPrinting disk usage...")?;
+    // if os.trim() == "Linux" {
+    //     run_cmd!(sudo docker exec paradedb du -bcs /bitnami/postgresql/data)?;
+    // } else {
+    //     run_cmd!(docker exec paradedb du -bcs /bitnami/postgresql/data)?;
+    // }
+
+    // run_cmd!(echo "\nPrinting results...")?;
+    // let file = File::open("log.txt")?;
+    // let reader = io::BufReader::new(file);
+    // let mut i = 0;
+
+    // for line in reader.lines() {
+    //     let line = line?;
+    //     if let Some(time_str) = line.split("Time: ").nth(1) {
+    //         if let Some(time_str) = time_str.split(" ms").next() {
+    //             let time: f64 = time_str.parse().unwrap_or(0.0);
+    //             if i % 3 == 0 {
+    //                 print!("[");
+    //             }
+    //             print!("{}", time / 1000.0);
+    //             if i % 3 != 2 {
+    //                 print!(",");
+    //             } else {
+    //                 println!("],");
+    //             }
+    //             i += 1;
+    //         }
+    //     }
+    // }
+    Ok(())
+}
+
+async fn run_queries(conn: &mut PgConnection) -> Result<()> {
+    let tries = 3;
+    let os = run_fun!(uname)?;
+    env::set_var("PGPASSWORD", "postgres");
+
+    for query in run_query_sql().lines() {
+        if query.trim().is_empty() {
+            continue;
+        }
+
+        // Clear the cache on Linux systems
+        if os.trim() == "Linux" {
+            run_cmd!(sync)?;
+            run_cmd!(echo 3 | sudo tee /proc/sys/vm/drop_caches)?;
+        }
+
+        println!("{}", query);
+
+        for _ in 1..=tries {
+            // Start timing
+            let start = Instant::now();
+
+            // Execute the query
+            conn.execute(query).await.unwrap();
+
+            // Calculate elapsed time
+            let duration = start.elapsed();
+
+            // Print the elapsed time in milliseconds
+            println!("Time: {:.3} ms", duration.as_secs_f64() * 1000.0);
+        }
+    }
+
+    Ok(())
+}
+
+fn create_query_sql(file_path: &str) -> String {
+    format!(
+        r#"
+DROP EXTENSION IF EXISTS pg_lakehouse CASCADE;
+CREATE EXTENSION IF NOT EXISTS pg_lakehouse;
+DROP FOREIGN DATA WRAPPER IF EXISTS local_file_wrapper CASCADE;
+DO $$
```

Comment:
if you drop the foreign data wrapper you can remove these checks for if the wrapper/server exist

Reply:
I don't think running benchmarks should drop anything. I think we should remove the `DROP FOREIGN DATA WRAPPER` and keep the checks.

Reply:
The problem I had was that an old version of the table was in my postgres and it was "blocking" development before I realized that was the case. Benchmark runs should be idempotent IMO, each should revert the state of the postgres and make sure to start from a clean state, no?

Reply:
Hmm realizing that Phil added this on purpose though, so I'll keep it in and remove the checks.